### ISSUE TITLE
Running Simulations from Linux terminal

### ISSUE THREAD

**Issue by umairkhawaja** [Unity-Technologies/com.unity.perception#487]
Hi, is it possible to define a FixedLengthScenario JSON configuration, and then pass it as an argument to run a simulation on a linux terminal?

**Comment by StevenBorkman** [comment]
Yes, that should be possible.

We support the following command line arguments:
* --scenario-config-file: Execute the player with the scenario configuration file (for example, --scenario-config-file=path/to/your/scenarioConfiguration.json).
* --output-path: Define the output of a standalone player with an '--output-path' command line argument (for example, --output-path path/to/your/output). If you do not use this command, the default output location would be at the [local persistent data directory](https://docs.unity3d.com/ScriptReference/Application-persistentDataPath.html)

**Comment by umairkhawaja** [comment]
Hi, thanks for your reply. I have never ran unity on terminal, so could you additionally help with the entire bash command?
I built my project for linux, and created an executable but when I ran it, it just opened to a black screen and then got stuck

**Comment by umairkhawaja** [comment]
@StevenBorkman Hi, can you help figuring out the command please?

**Comment by StevenBorkman** [comment]
Sorry I haven't replied yet. I'm reaching out to other team members who have access to a linux build to get more information, but from what I believe, the command should be as simple as:
```
./path/to/your/build/project_build_executable --scenario-config-file=/path/to/your/config/scenarioConfiguration.json
```

Is this similar to what you are using?

Also have you checked your build log to see if there are any issues with the build?

**Comment by umairkhawaja** [comment]
Yes this is what I used, but this is what I see after running the command. There are no build errors on the console of Unity Editor and the build finishes
![Screenshot from 2022-05-24 12-43-30](https://user-images.githubusercontent.com/13720047/169976784-4bdd7dab-fe70-4e4d-b9ce-7ebca8c5f9f3.png)

**Comment by umairkhawaja** [comment]
Hi @StevenBorkman, do you have any updates on the issue?

**Comment by RuiyuZ** [comment]
Hi @umairkhawaja, can you try appending `> output.txt` to the end of the command and see if there are any error logs being written to the output.txt file?

**Comment by umairkhawaja** [comment]
Hi,

Nothing is written to the output file
